File: DSA/Manav Rajput/Day1_Question_2.java
import java.util.Scanner;

class number
{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n, k, count = 0;
        System.out.println("Enter n:");
        n = sc.nextInt();
        System.out.println("Enter array of integers:");
        int arr[] = new int[n];
        for(int i = 0; i < n; i++)
        {
            arr[i] = sc.nextInt();
        }
        System.out.print("k = ");
        k = sc.nextInt();
        
        for(int i = 0; i < n; i++)
        {
            for(int j = i + 1; j < n; j++)
            {
                int sum = arr[i] + arr[j];
                if(sum % k == 0)
                {
                    count++;
                }
            }
        }
        System.out.println("Number of pairs = " + count);
    }
}

File: DSA/Arshdeep-Kaur/Day1_Question2.cpp
#include <iostream>
using namespace std;

int main() {
    int arr[10], n, k, pairs=0;
    cout << "Enter the number of elements of the array: ";
    cin >> n;
    cout << "Enter the elements of the array: ";
    for (int i=0; i<n; i++) {
        cin >> arr[i];
    }
    cout << "Enter the value of k: ";
    cin >> k;

    for (int i=0; i<n; i++) {
        for (int j=0; j<n; j++) {
            if (arr[i] < arr[j]) {
                int sum = arr[i] + arr[j];
                if (sum % k == 0) {
                    pairs++;
                }
            }
        }
    }

    cout << pairs;
}
